example/rocketmq_v1alpha1_rocketmq_cluster.yaml

```diff
@@ -163,6 +163,28 @@ spec:
         spec:
           containers:
             - name: console
+              env:
+                - name: ROCKETMQ_CONFIG_LOGINREQUIRED
+                  value: "true"
               image: apacherocketmq/rocketmq-console:2.0.0
               ports:
-                - containerPort: 8080
\ No newline at end of file
+                - containerPort: 8080
+              volumeMounts:
+                - name: console-config
+                  mountPath: /tmp/rocketmq-console/data
```

Comment:
It seems a bit curious to place the configuration files under `/tmp`.

Reply:
It's default directory of console data, see https://github.com/apache/rocketmq-dashboard/blob/6456630324e3ff22217a6da221a64c09bea82a72/src/main/resources/application.yml#L51